src/Router.tsx

```diff
@@ -39,6 +41,16 @@ export default function Router() {
         <Route path="/projects">
           <Projects />
         </Route>
+
+        <Route path="/p/:ensName(*.eth)">
```

Comment:
this regex is very permissive and potentially not ideal, but i think we can rely on the Ethers ENS resolver to fail if dodgy input is here.